### PR TITLE
Makes it possible to wrench barrel lids open/close.

### PR DESCRIPTION
**Pull request by colossusqw**
[LABEL][QoL][science]

## About the PR 
This PR makes it so chemical barrels can have their lids opened and closed with a wrench.

## Why's this needed? 
This is to avoid unintentional chemical cookoffs in public spaces.

```changelog
(u)colossusqw
(+)Chemical barrels can now be wrenched open/closed to avoid horrible chemical leaks.
```

### CODE REVIEW

File: code/modules/chemistry/tools/dispensers.dm [hunk 1]
```diff
@@ -426,6 +426,16 @@ TYPEINFO(/obj/reagent_dispensers/watertank/fountain)
 			src.name = t
 
 			src.desc = "For storing medical chemicals and less savory things."
+
+		if (istool(W, TOOL_WRENCHING))
+			if(src.flags & OPENCONTAINER)
+				user.visible_message("<b>[user]</b> wrenches the [src]'s lid close!")
```
Comment (Tarmunora) [inline code/modules/chemistry/tools/dispensers.dm:432]:
```suggestion
				user.visible_message("<b>[user]</b> wrenches the [src]'s lid closed!")
```

**Comment by BatElite** [comment]
Seems like this would benefit from sprites to differentiate the two states

**Comment by colossusqw** [comment]
sprites for the open lid barrels
image
image
image
image

**Comment by colossusqw** [comment]
sprites for the open lid barrels
image
image
image
image

**Pull request merged**